### PR TITLE
simplify group assignment, use for session id

### PR DESCRIPTION
**Pull request by malaffoon**
The legacy group feature was a fairly complex thing that generated random groups by section and staff. It resulted in some fairly bizarre groupings and was a bit confusing. So i simplified it: groups are created by grade and subject; using the school size, it results in a few groups per grade. Those groups are then used as the session for a test opportunity.

The upshot is every student will be in two groups per testing grade they participate in. Every opportunity will now have a sessionId that is the same as the group name for that subject. For example, Alice might have groups: ELA-G3-100, Math-G3-200, ELA-G4-5700, Math-G4-6100. Her test results will have sessionIds with those same values. If the sessionId being the same as the group name is confusing i suppose i could hash the value or something.

### CODE REVIEW

File: data_generator/sbac_generators/population.py [hunk 1]
```diff
@@ -114,34 +118,36 @@ def repopulate_school_grade(school: School, grade, grade_students, id_gen, state
...
-
-        groups_for_student = random.sample(all_groups, len(ns_for_student))
-
-        for n, group in zip(ns_for_student, groups_for_student):
-            setattr(grade_student, "group_%i_id" % (n,), group.id)
-            setattr(grade_student, "group_%i_text" % (n,), group.name)
+    # generate lists of subgroups for each subject corresponding to school's group size
+    num_groups = int(ceil(len(grade_students) / school.group_size))
+    groups = []
+    for subject in subjects:
```
Comment (malaffoon) [inline data_generator/sbac_generators/population.py]: not particularly pythonic but it works ...

File: scripts/submit_xml.sh [hunk 2]
```diff
@@ -9,9 +9,9 @@ pushd "$SCRIPT_DIR/../out"
 
 # fetch bearer token (need jq installed)
 HOST="http://localhost:8080"
-ACCESS_TOKEN=`curl -s -X POST --data "grant_type=password&username=dwtest@example.com&password=password&client_id=ets&client_secret=ets123" https://sso-deployment.sbtds.org:443/auth/oauth2/access_token?realm=/sbac | jq -r '.access_token'`
-# if host is using the "stub" token service, this works:
-# ACCESS_TOKEN="sbac;dwtest@example.com;|SBAC|ASMTDATALOAD|CLIENT|SBAC||||||||||||||"
+ACCESS_TOKEN=`curl -s -X POST --data "grant_type=password&username=dwtest@example.com&password=password&client_id=sbacdw&client_secret=sbacdw123" https://sso-deployment.sbtds.org:443/auth/oauth2/access_token?realm=/sbac | jq -r '.access_token'`
```
Comment (malaffoon) [inline scripts/submit_xml.sh]: unrelated

**Comment by agorina** [comment]
What would be an average group size? 
I think we have agreed that the max is 200 students. Just want to make sure we have group sizes close to this number.

**Comment by malaffoon** [comment]
The group sizes should range from about 10-50 now.
